### PR TITLE
Readme cds update

### PR DESCRIPTION
**Pull request by robertblincoe**
This is Robert, not Matt. Update to ReadMe.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ git config --global user.email "Your.Name@levellingup.gov.uk"
 [python_extension]: https://marketplace.visualstudio.com/items?itemName=ms-python.python
 [configure_python_interpreter]: https://code.visualstudio.com/docs/python/python-tutorial#_select-a-python-interpreter
 
-### Running the application
+### Running the application
```
Comment (MattSutton17) [inline README.md:53]: Very minor, appear to have added some trailing whitespace here.

File: README.md [hunk 2]
```diff
@@ -100,11 +100,26 @@ echo "Remember to run Black and Pylint before pushing to GitHub"
...
-1. Enable [Export headers option][export-headers] and then restart the SQL Server Management Studio.
+1. Check data available. From within the DAP AWS Workspace, open "Microsoft SQL Server Management Studio 18".
 1. Enter the Server name as "DAP-SQLTEST\CDS", and under Options specify the Database name as "Dashboards".
-1. Open the "New Query" from the Toolbar, paste in the query that corresponds to your table and execute.
-1. Right click on the results set and select "Save Results as" to an appropriate place in the `data/` directory.
+1. Open the "New Query" from the Toolbar, paste in the query that corresponds to your table or view and execute.
+1. In the data folder of your dashboard create new file. Import pandas and pyodbc.
+1. Enter code:
+``` 
+    conn = pyodbc.connect(
```
Comment (MattSutton17) [inline README.md:109]: Maybe get rid of this first tab to help with readability?

**Review by MattSutton17 — APPROVED** [review]
Nice update, helps to spell things out.

**Review by kateriley1 — APPROVED** [review]
Looks great  and very clear.

**Review by backslashbaker — APPROVED** [review]

**Pull request merged**